docs/user-guide/solver/08-hybrid-studies.md

```diff
@@ -0,0 +1,91 @@
+# Hybrid studies
+
+_**This feature is under development**_   
+
+
+Antares-Solver now allows conducting "hybrid" (solver x [modeler](../modeler/01-overview-modeler.md)) studies.  
+This enables using [antares-solver studies](02-inputs.md), in which the simulated system is enriched with components 
+that are described in the [antares-modeler format](../modeler/05-model.md).
+
+## Input structure
+To define a hybrid study, define a [solver study](02-inputs.md), and simply add the modeler [files and directories](../modeler/02-inputs.md) 
+to the input directory.    
+The parameter.yml file from modeler studies is not needed (if it exists, it will be ignored). The [solver parameters](04-parameters.md) 
+are used, since hybrid studies are conducted using [antares-solver](10-command-line.md).  
+You can find simple examples by checking out the studies used in our [test base](https://github.com/AntaresSimulatorTeam/Antares_Simulator/blob/develop/src/tests/cucumber/features/solver-features/hybrid_studies.feature).  
+
+## Output files
+The hybrid simulation outputs the [same files as a legacy study](03-outputs.md). But these files only contain the 
+optimization results of the system components that were created by the legacy study.  
+
+The optimal values of the variables created by the modeler (ie the state of the components modelled in the 
+[system file](../modeler/02-inputs.md#system-file)) are output in separate files.  
```

Comment:
outputted ?

Reply:
both are OK but output is more widely used (past tense of put is put)